### PR TITLE
DotNet Transactions doc updates for 1.1

### PR DESCRIPTION
**Pull request by RiPont**
Release notes.

Query support.

Custom Metadata Collections.

### CODE REVIEW

**Comment by RiPont** [comment]
TXNN-112

File: modules/howtos/examples/TransactionsExample.cs [hunk 1]
```diff
@@ -135,6 +134,8 @@ async Task Examples()
                     var docC = await ctx.GetAsync(_collection, "doc-c").ConfigureAwait(false);
                     await ctx.RemoveAsync(docC).ConfigureAwait(false);
 
+                    // This call is optional - if you leave it off, the transaction
+                    // will be committed anyway.
                     await ctx.CommitAsync().ConfigureAwait(false);
                 }).ConfigureAwait(false);
```
Comment (osfameron) [inline modules/howtos/examples/TransactionsExample.cs:140]: Not added in this commit, I realise, but I'd previously noticed that https://devblogs.microsoft.com/dotnet/configureawait-faq/#when-should-i-use-configureawaitfalse suggests:

> This leads to the general guidance of: if you’re writing app-level code, do not use `ConfigureAwait(false)`.

I'm still a little confused about whether we need these in our examples? (e.g. I'd assume we *do* need it in the SDK library code, but not the docs examples?)

File: modules/howtos/examples/TransactionsExample.cs [hunk 2]
```diff
@@ -471,6 +472,195 @@ async Task CompleteLogging()
             // #end::full-logging[]
         }
 
+                async Task QueryExamples()
```
Comment (osfameron) [inline modules/howtos/examples/TransactionsExample.cs]: Indentation

File: modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc [hunk 3]
```diff
@@ -132,7 +137,36 @@ include::example$TransactionsExample.cs[tag=examples,indent=0]
...
+=== Removing
+
+As with replaces, removing a document requires awaiting a `TransactionGetResult` from a previous transaction operation first.
+
+[source,csharp]
+----
+include::example$TransactionsExample.cs[tag=remove,indent=0]
+----
+
+== Key-Value Reads
```
Comment (osfameron) [inline modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc:169]: This probably needs a `=== Getting` as we've removed the level 2 header about that, so if someone is scanning the list of gerunds like "Inserting", "Removing", ... they find what they're looking for.

(As we removed the level 2 `== Getting documents` above)

**Review by osfameron — COMMENTED** [review]
Couple of comments which I'll garden, looks good.

File: modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc [hunk 4]
```diff
@@ -152,35 +188,102 @@ include::example$TransactionsExample.cs[tag=getReadOwnWrites,indent=0]
...
 
 [source,csharp]
 ----
-include::example$TransactionsExample.cs[tag=replace,indent=0]
+include::example$TransactionsExample.cs[tag=queryExamplesUpdate,indent=0]
 ----
 
-=== Removing
+And an example combining SELECTs and UPDATEs.
+It's possible to call regular Java methods from the lambda, as shown here, permitting complex logic to be performed.
```
Comment (programmatix) [inline modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc:221]: You can call Java methods from .Net now? Wowzers!

Comment (osfameron) [reply modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc:221]: Eek, will garden, thanks for spot.

Comment (osfameron) [reply modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc:221]: https://github.com/couchbase/docs-sdk-dotnet/pull/224

**Pull request merged**